pages/HOME.md

```diff
@@ -1,69 +1,88 @@
-  		<input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button">
-  		<span id="mce-EMAIL-HELPERTEXT" class="helper_text"></span>
-  	</div>
-  	<div id="mce-responses" class="clear">
-  		<div class="response" id="mce-error-response" style="display:none"></div>
-  		<div class="response" id="mce-success-response" style="display:none"></div>
-  	</div>
-  	<div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_8befe3303cf82796d2c1a1aff_271812000b" tabindex="-1" value=""></div>
-  </div>
-</form>
-</div>
-<!-- /mailchimp -->
+# Welcome to Gno.land
+
+---
+
+## An interpretation of the Golang (Go) programming language for advanced developers and intrepid pioneers to build succinct, composable smart contracts for social coordination.
+
+If you’re concerned about information censorship and want to contribute to the #GnoWorldOrder, follow our socials to find out how.
+
+Gno.land is in building mode. If you want to help lay the foundations of a fairer and freer world through innovative ideas and exceptional code, join us today.
+
+<div class="stack">
+
+[About Gno.land](/about)
+[Explore Github repo](https://github.com/gnolang)
+
+</div><!-- end stack-->
+
+<div class="jumbotron">
+
+### Game of Realms
+
+The first high-stakes contest will see participants compete for tiered membership to co-own the Gno.land blockchain. A series of complex technical and non-technical tasks will challenge contributors to create innovative patterns that push the chain to new limits. Start building the foundation for tomorrow through key smart contracts and other contributions that change our understanding of the world.
+
+<div class="stack">
+
+[Join Game Of Realms](/game-of-realms)
+
+</div><!-- end stack-->
+</div><!-- end jumbotron-->
+
+### Fund and Grants Program
+
+Are you a builder, tinkerer, or researcher? If you’re looking to create awesome dApps, tooling, infrastructure, or smart contract libraries on Gno.land, you can apply for a grant. The Gno.land Ecosystem Fund and Grants program provides financial contributions for individuals and teams to innovate on the platform.
+
+<div class="stack">
+
+[More information here](https://github.com/gnolang/ecosystem-fund-grants)
+
+</div><!-- end stack-->
+
+<div class="columns-3">
+<div class="column">
+
+### Learn about Gno.land
+
+- [About](/about)
+- [Subscribe](#subscribe)
+- [Tokenomics (soon)](#)
+- [Blogs](/r/gnoland/blog)
+- [Events](/events)
+- [Partners (soon)](#)
+
+</div><!-- end column-->
+
+<div class="column">
+
+### Build with Gnolang
+
+- [Gno dev with CLI (soon)](#)
+- [Explore the Universe](/ecosystem)
+- [Test in the browser (soon)](#)
+- [About the Gnolang Language](/gno-language)
+- [Docs/ Tutorials](https://github.com/gnolang)
+- [Gno by example](https://gno-by-example.com/)
+- [Getting started video (soon)](#)
+
+</div><!-- end column-->
+<div class="column">
+
+### Explore the universe
```

Comment:
we should add a link to current (and previous) testnets; in this section or in a dedicated one